### ISSUE TITLE
Is it possible when changing the value of one slider to automatically change the value of another by a coefficient?

### ISSUE THREAD

**Issue by ytiruces** [soundar24/roundSlider#128]
Hi!

Is it possible when changing the value of one slider to automatically change the value of another by a coefficient?

TY.

**Comment by soundar24** [comment]
@ytiruces are you expect something like this?

https://jsfiddle.net/soundar24/3qec1w0v/

**Comment by ytiruces** [comment]
@soundar24  Yes, but I need them to be codependent. If I change the value of the second slider, then the first one will also change the value. And vice versa.
